tmgmt_thebigword.module

```diff
@@ -31,6 +31,12 @@ function tmgmt_thebigword_menu() {
       'access callback' => TRUE,
       'type' => MENU_CALLBACK,
     ),
+    'admin/tmgmt/overview/tmgmt_thebigword_pull_all_remote' => array(
+      'title' => 'Pull all thebigword translations',
+      'page callback' => 'tmgmt_thebigword_pull_all_remote',
+      'access arguments' => array('administer tmgmt+accept translation jobs'),
```

Comment:
drupal 7 doesn't have that feature. it only works for you as admin because it's just treated as a single permission.

To support both permissions, you need an access callback that checks user_access() || user_access()